miner/worker.go

```diff
@@ -78,6 +78,14 @@ const (
 )
 
 var (
+	inturnBlocksCount              = 0
```

Comment:
Does it have concurrent issue? How about add it as Gauge, and calculate in grafana.

```
bidExistGauge             = metrics.NewRegisteredGaugeFloat64("worker/bidExist", nil)
bidWinGauge               = metrics.NewRegisteredGaugeFloat64("worker/bidWin", nil)
InturnBlockGauge             = metrics.NewRegisteredGaugeFloat64("worker/inturnblock", nil)

// usage
bidExistGauge.Mark(1)
bidWinGauge.Mark(1)
InturnBlockGauge.Mark(1)
```